src/api/app/models/bs_request.rb

```diff
@@ -1169,10 +1169,10 @@ def forward_to(project:, package: nil, options: {})
                  source_package: action.target_package,
                  source_rev:     rev,
                  target_project: project,
-                 target_package: package }
+                 target_package: package,
+                 type:           'submit' }
```

Comment:
Why is the type now submit?

Reply:
This was [previously](https://github.com/openSUSE/open-build-service/pull/4872/files#diff-a1bb0bdb9ac85a32ba6d96c4289745fcL428) done via
```ruby
  action = BsRequestActionSubmit.new(opts)
```